### DISCUSSION TITLE
Packing up March [April Check-in]

### DISCUSSION THREAD

**Discussion by grandmas-favorite** [community/community#120747] [Packages]
![Header for the Packages category shows a dark purple abstract background with white text reading “Packages Check-in”. In the foreground there is an image of a person holding a magnifying glass next to a cartoon representation of a packing box. To the right, there is a GitHub logo in two-tone purple.](https://github.com/community/community/assets/144172738/86d969aa-6fdd-4e23-bbf2-ea5459f18e51)

Hello Community! 👋 

April’s about halfway through, so it's high time for a Packages check-in!

### 🏆 Excellent in Packages: shout out to these wonderful community experts 👏: 

Shoutout to @nt-ergon for [this killer workaround](https://github.com/orgs/community/discussions/102983) for an issue that @roerohan was facing with the Packages UI. Such a clever idea to weave around the issue and fix it from the console!

Round of applause for @berry120’s [nifty workaround](https://github.com/orgs/community/discussions/119679) for an issue that a few Community members were experiencing with authentication in Packages. Nothing better than a tl:dr quick fix!

### 🗳️ Packages Info: All updates, info, releases, etc. happening in the land of Packages!

The Latest Packages Update: The Packages category is steady, but in case you missed it back in September, we announced changes to permissions for packages. We are restricting the [refs REST API endpoint](https://docs.github.com/rest/git/refs#create-a-reference) from accepting POSTs from users and apps that only have the [permission to read and write packages](https://docs.github.com/packages/learn-github-packages/configuring-a-packages-access-control-and-visibility). Previously, this endpoint accepted updates to both tags and branches. More on this in our [changelog blog](https://github.blog/changelog/2023-09-26-changes-to-token-permission-on-packages/).

### 🪐 GitHub Universe 2024: Read about the event and take advantage of the 35% off sale ending July 8th!

[GitHub Universe 2024](https://github.blog/2024-04-16-the-worlds-fair-of-software-join-us-at-github-universe-2024/): As the world’s fair of software, GitHub Universe 2024 will be an unparalleled gathering of the brightest minds, companies, and innovators in the industry!

### 📎For all Developers in and out of the Packages category:

GitHub Certifications are generally available: Unlock your full potential with GitHub Certifications! Earning a GitHub certification will give you the competitive advantage of showing up as a GitHub expert. [Learn more on our blog](https://github.blog/2024-01-08-github-certifications-are-generally-available/).

**Comment by JorenVsyntra** [comment]
Hey everyone!
